server/service/integration_core_test.go

```diff
@@ -6602,7 +6602,7 @@ func (s *integrationTestSuite) TestListSoftwareAndSoftwareDetails() {
 	// create a bunch of software
 	sws := make([]fleet.Software, 20)
 	for i := range sws {
-		sw := fleet.Software{Name: "sw" + strconv.Itoa(i), Version: "0.0." + strconv.Itoa(i), Source: "apps"}
+		sw := fleet.Software{Name: fmt.Sprintf("sw%02d", i), Version: "0.0." + strconv.Itoa(i), Source: "apps"}
```

Comment:
That's a fix for the now-non-deterministic order of software ingestion, I had it already in another PR but had to copy it here to give CI a chance.